Sources/NIOWebSocket/Base64.swift

```diff
@@ -33,36 +33,29 @@ internal struct Base64 {
   static func encode<Buffer: Collection>(bytes: Buffer)
     -> String where Buffer.Element == UInt8
   {
+    guard bytes.isEmpty == false else {
```

Comment:
Rather than an explicit comparison to `false` can this just be `guard !bytes.isEmpty`?